tests/test_cli.py

```diff
@@ -36,11 +36,11 @@ def test_cli_invalid_config_file_path():
 # ----------------------------------------------------------------------------------------------------
 def test_cli_config_from_file():
     # Act
-    actual_args = cli(["-c=tests/data/config.yml", "--regenerate", "--no-gui", "-vvv"])
+    actual_args = cli(["-c=tests/data/config.yml", "--regenerate", "-vvv"])
```

Comment:
We should pass the `--gui` field to test if passing it results in `actual_args.gui = True`
```suggestion
    actual_args = cli(["-c=tests/data/config.yml", "--regenerate", "--gui", "-vvv"])
```